packer/scripts/firstuser.sh

```diff
@@ -86,6 +95,8 @@ checkFirstUser() {
    echo "${status}"
 }
 
+stopServices
+cleanServicesPIDs
```

Comment:
After installation when when ```service rackhd start``` if there's any external environment problem, such as "no mongo or rabbit mq service" or there's no enough network card, then the "service start" will fail and left PID files. After env repair when start again the PID files will stop service start. So they must be deleted.

In this case when rackhd start service eht1 has not been setup yet.